src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -280,6 +291,31 @@ private async void DiagnosticService_DiagnosticsUpdated(object sender, Diagnosti
             }
         }
 
+#pragma warning disable VSTHRD100 // Avoid async void methods
+        private async void RequestExecutionQueue_Errored(object sender, RequestShutdownEventArgs e)
+#pragma warning restore VSTHRD100 // Avoid async void methods
+        {
+            // Since this is an async void method, exceptions here will crash the host VS. We catch exceptions here to make sure that we don't crash the host since
+            // the worst outcome here is that guests don't get the log message. We still want to ensure the connection is closed though.
+            try
+            {
+                // log message and shut down
+
+                var message = new LogMessageParams()
+                {
+                    MessageType = MessageType.Error,
+                    Message = e.Message
+                };
+
+                await _jsonRpc.NotifyWithParameterObjectAsync(Methods.WindowLogMessageName, message).ConfigureAwait(false);
+            }
+            catch (Exception ex) when (FatalError.ReportWithoutCrash(ex))
+            {
+            }
+
+            await ExitAsync(/* cancellationToken: */ default).ConfigureAwait(false);
```

Comment:
ExitAsync doesn't use its cancellation token, so the named parameter for this is `_`, hence this style

Reply:
I'd add that comment in the code as otherwise anybody else looking at this will wonder why we didn't use named parameters.

Reply:
Inline parameter hints is going to make these types of things more prevalent.. when looking in the IDE its immediately obvious, if you ask me :)

![image](https://user-images.githubusercontent.com/754264/91390294-b0eafb80-e87c-11ea-8168-31b9a6bab7d9.png)

Will add a comment for lesser IDEs, like GitHub :P